tox.ini

```diff
@@ -2,16 +2,20 @@
 # See LICENSE.rst for details.
 
 [tox]
-envlist = py{35,36,37,38},qa,doc
+envlist = py{35,36,37,38},qa,doc,watch
 skip_missing_interpreters = True
 
 [testenv]
 commands =
     python setup.py install
-    coverage run -m py.test -v -r wsx -s
+    coverage run -m py.test -v -r wsx -s {posargs}
     coverage report
 deps = .[test]
 
+[testenv:watch]
```

Comment:
what does watch do? can you add a comment here with short description?

Reply:
allows you to run `tox -e watch` which in turn runs pytestwatch which will automatically run tests when files change. 

Obvs. this is useful for development only.

Reply:
"use pytestwatch to run tests when files change" sounds like a good comment to me

Reply:
done